tethys_portal/templates/tethys_portal/accounts/lockout.html

```diff
@@ -0,0 +1,44 @@
+{% extends "page.html" %}
+{% load static humanize %}
+
+{% block title %}Locked Out!{% endblock %}
+
+{% block styles %}
+{{ block.super }}
+<style>
+    body {
+      background: {{ site_globals.background_color|default:'#ffffff' }};
+    }
+</style>
+<link href="{% static '/tethys_portal/css/lockout.css' %}" type="text/css" rel="stylesheet" />
+{% endblock %}
+
+{% block flash %}{% endblock %}
+
+{% block primary_content %}
+  <div class="lockout-container">
+    <div class="row">
+      <div class="col-lg-6">
+        <div class="lockout-image-container center-parent">
+          {% block lockout_image %}
+            <img class="centered" src="{% static 'tethys_portal/images/lockout.png' %}">
+          {% endblock %}
+        </div>
+      </div>
+      <div class="col-lg-6">
+        <div class="row">
+          <div class="col-lg-12">
+            <div class="lockout-message-container">
+              <p>You have exceeded the amount of allowed login attempts ({{ failure_limit }}).</p>
+              {% if cooloff_time %}
+                <p>Your account will be unlocked {{ cooloff_time|human_duration }}, after which you may try again.</p>
```

Comment:
This is nit-picky, but it would make more sense to me to say "Your account will be _locked for_ 30 min, after which you may try again."

I'm assuming the humanize thing makes it say _in 30 min_, which is fine, but in that case I wonder if it would sound better to say: "Your account will be unlocked in 30 min, _at which point you may try again._" 

If you disagree then go ahead and merge as is.

Reply:
@sdc50 thanks for the suggestion. I modified it to match your second suggestion because as you surmised, humanize returns "in 30 minutes".